lib/codegen/ApplyPasses.cpp

```diff
@@ -73,7 +80,7 @@ void generate_object_file(llvm::Module &mod, llvm::raw_ostream &os) {
   }
 #endif
 
-  auto triple = sys::getDefaultTargetTriple();
+  auto triple = "@BACKEND_TARGET_TRIPLE@";
```

Comment:
I'm starting to think this should maybe be in config/macros.h. what do you think?

Reply:
Addressing in a followup PR.